openchallenges/fargate_cpu_memory.py

```diff
@@ -0,0 +1,71 @@
+from enum import Enum
+
+
+class FargateCpuMemory(Enum):
```

Comment:
i'm not a fan of this because these combinations are already kept in AWS and they could change over time.  Keeping a copy here would introduce a maintenance burden of keeping this up to date.  Instead how about we just query AWS for the correct combination and if the user doesn't provide the right combination throw an exception and provide a link for the user to lookup valid combos in AWS?